tests/models/language/generation/test_hybrid.py

```diff
@@ -72,12 +102,21 @@ def test_models(
             example_prompts, max_tokens, num_logprobs)
 
     if model in V1_SUPPORTED_MODELS:
+        if model in HYBRID_MODELS and model in ATTN_BLOCK_SIZES:
+            block_size = ATTN_BLOCK_SIZES[model]
+        else:
+            block_size = 16
+
         with monkeypatch.context() as m:
             m.setenv("VLLM_USE_V1", "1")
+            if model in HYBRID_MODELS:
+                # required due to reorder_batch behaviour
+                m.setenv("VLLM_ATTENTION_BACKEND", "FLASHINFER")
```

Comment:
We should probably throw if the user tries to use unsupported attention backend (e.g., one that does not reorder batch in same way as Flashinfer).

Reply:
Yes, will add a TODO. I see now that this logic could easily be made more flexible, but let's do it as a followup since we need FlashInfer for other reasons currently.